### PR TITLE
Refactor cmd/ to have less coupling

### PR DESCRIPTION
**Pull request by bcrochet**
No more inits are being used in the individual command files.
The one that remains is to make sure that initConfig is called 
when cobra is initialized.

This should increase coverage slightly, and make the commands less
interdependent.

Signed-off-by: Brad P. Crochet <brad@redhat.com>

### CODE REVIEW

File: cmd/support_container_test.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// go:test !race
```
Comment (acornett21) [inline cmd/support_container_test.go:1]: Why does the race detection need to be ignored?

File: cmd/support_operator_test.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
+// go:test !race
```
Comment (acornett21) [inline cmd/support_operator_test.go:1]: Same question as above.

File: cmd/support_container_test.go (hunk 1 above)
Comment (bcrochet) [reply cmd/support_container_test.go:1]: This was added by @komish to the support.go. I don't know why though.

File: cmd/support_operator_test.go (hunk 2 above)
Comment (bcrochet) [reply cmd/support_operator_test.go:1]: Same answer as above. :)

File: cmd/support_container_test.go (hunk 1 above)
Comment (acornett21) [reply cmd/support_container_test.go:1]: @komish Is this some leftover from `promptui`? Or is it still needed?

Comment (komish) [reply cmd/support_container_test.go:1]: Yep, this can be removed. PromptUI leftover.

File: cmd/support_operator_test.go (hunk 2 above)
Comment (komish) [reply cmd/support_operator_test.go:1]: Same answer as above!

File: cmd/check.go [hunk 3]
```diff
@@ -13,13 +13,13 @@ import (
 	"github.com/spf13/viper"
 )
 
-var checkCmd = &cobra.Command{
-	Use:   "check",
-	Short: "Run checks for an operator or container",
-	Long:  "This command will allow you to execute the Red Hat Certification tests for an operator or a container.",
-}
+func check() *cobra.Command {
```
Comment (komish) [inline cmd/check.go:16]: Can we use the naming `<action>Cmd` for all functions that return cobra.Command? I won't go through and mark them all.

**Review by komish — CHANGES_REQUESTED** [review]
Just one requested change, otherwise looks good. 

ALSO I totally see at a support command refactor here that could have been a separate PR :P

We already discussed offline, but I generally don't like our use of functions to return cobra commands here. I know this pattern can be seen in various other projects, but _typically_ I see it in cases where the function's parameters might change the resulting cobra command. I'm not particularly a fan of _that_ approach either, but I understand what drives it. 

All that said: if it works, it works!

File: cmd/check.go (hunk 3 above)
Comment (komish) [reply cmd/check.go:16]: Can avoid using `new<action>Cmd` as I commonly see if you want, or use that instead if you prefer

Comment (bcrochet) [reply cmd/check.go]: Used the 'Cmd' suffix. Good suggestion.

**Review by acornett21 — APPROVED** [review]

**Pull request merged**